modules/rds/dev.tfvars

```diff
@@ -0,0 +1,10 @@
+identifier           = "himanshu-arc-rds"
+engine               = "MySQL"
+engine_version       = "5.7.44"
+allocated_storage    = "20"
+instance_class       = "db.t3.micro"
+username             = "postgres"
+password             = "postgres"
```

Comment:
you should use a random resource for generating password and store that password in ssm